components/__tests__/ReportList.test.js

```diff
@@ -0,0 +1,59 @@
+import { mount, flushPromises } from "@vue/test-utils";
+import ReportList from "../ReportList.vue";
+import { expect } from "vitest";
+import { VueQueryPlugin } from "@tanstack/vue-query";
+import { nextTick } from "vue";
+import { ShieldIcon } from "@axodotdev/fringe/lib";
+
+let wrapper;
+
+const createComponent = ({ props = {}, shallow = false } = {}) => {
+  wrapper = mount(ReportList, {
+    props,
+    shallow,
+    global: {
+      plugins: [VueQueryPlugin],
+    },
+  });
+};
+
+describe("ReportList component", () => {
+  beforeEach(async () => {
+    createComponent();
+    await flushPromises();
+
+    return () => {
+      wrapper.unmount();
+      wrapper = null;
+    };
+  });
+
+  const getOptions = () => wrapper.findAll('[data-test="dependency-option"]');
+
+  it("Shows correct list", async () => {
+    expect(wrapper.text()).toContain("clap");
+    expect(wrapper.text()).toContain("serde_json");
```

Comment:
**suggestion:** As in the other file, I would recommend testing the text via specific items rather than at the level of the `wrapper`.

Reply:
Done!

Reply:
I also do not see this reflected, although I totally believe you. Maybe part of changes that are still on your local machine?

Reply:
Actually I haden't, I confused the tests, it's done now, also made the contain be from the mock